packages/sdk/src/actor/behaviors/penBehavior.ts

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { BehaviorType, ToolBehavior } from '.';
+import { TransformLike } from '../..';
+
+
+/* export */ interface DrawData {
+	transform: TransformLike;
+	// Potentially additional data to come, such as:
+	// pressure: number;
+}
+
+/* export */ interface PenEventData {
+    drawData: DrawData[];
+}
+
+/**
+ * Pen behavior class containing the target behavior actions.
+ */
+/* export */ class PenBehavior extends ToolBehavior<PenEventData> {
```

Comment:
I would exclude `PenBehavior` and `ToolBehavior` from this PR. You really just need the 'performing' action on `ButtonBehavior`.

Reply:
The pen and tool code is already working and tested.  I don't see the issue with it being in this PR and not exported.  There are other teams that will want to use this and could simply export the class in their own fork to use it,  It is implemented in the test bed as well.

Reply:
We're going to learn a lot from all the planned changes to ButtonBehavior. Having these behaviors in the codebase might slow you down, or break them. Better to not have to maintain them in parallel if we're not shipping them yet.

Reply:
Making abstract as per our offline conversation.